kodilla-testing/src/main/java/com/kodilla/testing/user/SimpleUser.java

```diff
@@ -0,0 +1,13 @@
+package com.kodilla.testing.user;
+
+public class SimpleUser {
```

Comment:
A to co to?

Reply:
Przed zadaniem są polecenia utworzenia klasy SimpleUser - i wszystkie linijki, które się do niej odnoszą są wzięte z kursu. Zostawiłem ją, ponieważ przewija się ona też w późniejszych zadaniach.